core/job/spec.go

```diff
@@ -560,14 +562,31 @@ type WebhookSpec struct {
 	Endpoints []WebhookEndPoint
 }
 
-func NewAlertSpec(on string, channels []string, config Config) (*AlertSpec, error) {
+const (
+	DefaultSeverity  = "INFO"
```

Comment:
in here default severity is INFO, while in https://github.com/goto/optimus/pull/299/files#diff-ae8202ecdfcffa6355ad138f9caf0e43b8c5163a4a831e28be335bbcc3a66f2cR28 it is warning. Both should be warning, no?

Reply:
yes correct, 
fixing this